### PR TITLE
Docs: Update frontend.md

### PR DESCRIPTION
**Pull request by mthorning**
**What is this feature?**

Updates the frontend.md markdown file, removes incorrect lines which describe the type of case (camelCase, PascalCase, etc.) should be used.

**Why do we need this feature?**

This is a docs change to improve clarity.

**Who is this feature for?**

Contributors who are reading our style guide.

**Which issue(s) does this PR fix?**:
N/A

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [x] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: contribute/style-guides/frontend.md [hunk 1]
```diff
@@ -99,8 +99,6 @@ For directory names, use dash-case (sometimes called kebab-case).
 
 ### Functions
 
-Use PascalCase. For example:
-
 Use camelCase.
```
Comment (AgnesToulet) [inline contribute/style-guides/frontend.md:102]:
```suggestion
Use camelCase. For example:
```

File: contribute/style-guides/frontend.md [hunk 2]
```diff
@@ -147,8 +145,6 @@ type RequestInfo = ...
 
 ### Methods
 
-Use PascalCase. For example:
-
 Use camelCase.
```
Comment (AgnesToulet) [inline contribute/style-guides/frontend.md:148]:
```suggestion
Use camelCase. For example:
```

File: contribute/style-guides/frontend.md [hunk 3]
```diff
@@ -296,8 +292,6 @@ function transform(value?: string): TransformedValue | undefined {
 
 ### Variables
 
-Use PascalCase. For example:
-
 Use camelCase.
```
Comment (AgnesToulet) [inline contribute/style-guides/frontend.md:295]:
```suggestion
Use camelCase. For example:
```

**Review by AgnesToulet — APPROVED** [review]
LGTM! Just added some minor updates to stay consistent with the rest of the doc.

**Comment by mthorning** [comment]
> LGTM! Just added some minor updates to stay consistent with the rest of the doc.

Have applied the suggestions, thank you :)

**Comment by mthorning** [comment]
@AgnesToulet I'm not authorised to merge this PR, is that something you could do for me please?

**Pull request merged**